components/modal/index.en-US.md

```diff
@@ -0,0 +1,67 @@
+---
+type: Views
+category: Components
+english: Modal
+---
+
+Modal dialogs.
+
+## When to use
+
+When requiring users to interact with application, but don't want to jump to a new page to interrupt
+user's workflow, you can use `Modal` to create a new floating layer over currtent page for user 
+operations.
+
+In addition, if you need a simple confirm dialog to users, you can use `ant.Modal.confirm()`, 
+and so on.
+
+
+
+## API
+
+
+| Property       | Description           | Type             | Default       |
+|------------|----------------|------------------|--------------|
+| visible    | Determine whether a modal dialog is visible | Boolean          | no           |
+| confirmLoading | Determine whether to apply loding visual effect for OK button  | Boolean    | no           |
+| title      | A title of a modal dialog           | React.Element    | no           |
+| closable   | Determine whether a closable button is visible on top right of a modal dialog | Boolean    | true        |
+| onOk       | Specify a function that will be called when a user clicked OK button       | function     | no           |
```

Comment:
| onOk       | Specify a function that will be called when a user clicked OK button.       | function     | no           |